src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -68,9 +67,21 @@ class ErgoWalletActor(settings: ErgoSettings,
   // The state context is being updated by listening to state updates.
   private def stateContext: ErgoStateContext = storage.readStateContext
 
-  private var height: Int = ErgoHistory.GenesisHeight
+  /**
+    * Height of full-blocks chain as reported by the state
```

Comment:
```suggestion
    * Height of full-blocks chain (not the headers-chain) as reported by the state
```
Well, if you add `(not the headers-chain)` hit to the comments it will do the job.

Reply:
State can't report headers-chain height at all.

Reply:
You know about it, I don't. And the code is for me to read it. So I gave you a hint where I cannot read it.

Reply:
Please see a comment on inter-subjectiveness above